server/app.js

```diff
@@ -9,18 +9,22 @@ import cors from 'cors';
 import logger from './utils/logger';
 import authorize from './thirdparty/routes/authorize';
 import slackRouter from './thirdparty/routes/slackRouter';
+import githubRouter from './thirdparty/routes/githubRouter';
 import IntegrationService from './controller/IntegrationService';
+import AuthenticationMiddleware from './middlewares/authentication';
 
 dotenv.config();
 
 const app = express();
 const modelDirPath = path.join(__dirname, 'models');
 const routeDirPath = path.join(__dirname, 'routes');
+const publicFolder = path.join(__dirname, '../assets');
 
 app.use(pino());
 app.use(express.json());
 app.use(express.urlencoded({ extended: false }));
-app.use(express.static(path.join(__dirname, 'public')));
+app.use(express.static(publicFolder));
+// app.use('/api/v1', AuthenticationMiddleware);
 app.set('view engine', 'ejs');
 app.set('views', path.join(__dirname, '/thirdparty/views'));
 requireAll(modelDirPath);
```

Comment:
all require should happen before any logic. Ideally, they should be the first thing